runtime/binding-grpc-kafka/src/main/java/io/aklivity/zilla/runtime/binding/grpc/kafka/internal/config/GrpcKafkaWithProduceResult.java

```diff
@@ -140,6 +150,22 @@ public void headers(
         builder.item(this::method);
         builder.item(this::replyTo);
         builder.item(this::correlationId);
+        metadata.forEach(m -> builder.item(i -> metadata(i, m)));
+    }
+
+    private void metadata(
+        KafkaHeaderFW.Builder builder,
+        GrpcMetadataFW metadata)
+    {
+        int nameLen = metadata.nameLen();
+        int nameLenWithPrefix = nameLen + 5;
+        buffer.putBytes(5, metadata.name().value(), 0, nameLen);
```

Comment:
Let's extract a `META_PREFIX_LENGTH` constant for `5` here to improve readability.